src/Entry.hs

```diff
@@ -0,0 +1,15 @@
+module Entry
+    ( Entry(..)
```

Comment:
I learned that when we're exporting data types, we need to make sure to export all the stuff that comes with it like `index`, `date` and `message`!

Reply:
Usually, but not always! Sometimes you might want to hide the internals of your data type and only expose a "smart constructor".